### ISSUE TITLE
ADC1173 symbol in library Analog_ADC.lib is wrong

### ISSUE THREAD

**Issue by FrankBuss** [KiCad/kicad-symbols#1165]
Pin AIN is wrong, it is pin 19, not pin 27. There is no pin 27 in a TSSOP24 package.

**Comment by evanshultz** [comment]
Hi Frank,

It appears this is the first contribution to KiCad! Thanks for taking the time to contact the librarian team, and sorry that it's because you found a possible error.

You are absolutely right:
![image](https://user-images.githubusercontent.com/1936989/48971559-13d8a000-efd0-11e8-829a-ef35cfd05ca1.png)

Pin 19 should be named "VIN" according to the datasheet as well.

I've submitted a PR to fix this symbol at https://github.com/KiCad/kicad-symbols/pull/1167. I wanted to get this fix implemented right away so I went ahead and made the PR, but in the future if you find an issue you're welcome to submit a PR yourself, if you wish.

Thanks again!
